test/check-application

```diff
@@ -158,6 +165,60 @@ class TestApplication(testlib.MachineCase):
         b.wait_not_present("modal-dialog div #btn-img-deleteerror")
         b.wait_not_in_text("#containers-images", "alpine:latest")
 
+    def testDownloadImage(self):
+        b = self.browser
+        m = self.machine
+
+        # Create and start registry container
+        m.execute("podman run -d -p 5000:5000 --name registry registry:2")
+
+        # Add local insecure registry into resgitries conf
+        m.execute("echo \"{0}\" > /etc/containers/registries.conf && systemctl restart io.podman.socket".format(REGISTRIES_CONF))
```

Comment:
Restarting the socket is rather pointless. This should do `try-restart io.podman.service`, or just `stop` as it will just be started again on demand.